app/src/App.tsx

```diff
@@ -83,7 +83,7 @@ function App() {
       element: <ProtectedRoute><EducadoAdmin /></ProtectedRoute>,
     },
     {
-      path: "/NewInstitution",
+      path: "/educado_admin/newinstitution",
```

Comment:
I believe we should use kebab case in URLs

Reply:
Fixed, now they look like `/educado-admin/new-institution`.